pdr_backend/ppss/trader_ss.py

```diff
@@ -44,6 +44,14 @@ def allowed_tradetypes(self) -> List[str]:
     def sim_confidence_threshold(self) -> float:
         return self.d["sim_only"].get("confidence_threshold", 0.0)
 
+    @property
+    def stop_loss(self) -> float:
```

Comment:
I just realized: a clearer name for these settings is:
- `stop_loss` -> `stop_loss_percent`
- `take_profit` -> `take_profit_percent`

So update ppss.yaml, trader_ss, etc accordingly. Thanks.